libs/api/domains/consultation-portal/src/lib/documents/documents.resolver.ts

```diff
@@ -0,0 +1,13 @@
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { DocumentService } from './documents.service'
+import { DocumentInfoResult } from '../models/documentInfoResult.model'
+
+@Resolver(() => DocumentInfoResult)
+export class DocumentResolver {
+  constructor(private documentService: DocumentService) {}
+
+  @Query(() => [DocumentInfoResult], { name: 'consultationPortalDocument' })
+  async getDocument(@Args('documentId') documentId: string): Promise<void> {
+    return this.documentService.getDocument(documentId)
+  }
```

Comment:
What kind of documents are these? PDFs? If so we might want to move it to our download service.